lib/proto/file.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     check: function (link, opts, callback) {
         let filepath = decodeURIComponent(url.parse(link, false, true).path || '');
         if (!path.isAbsolute(filepath)) {
-            const basepath = url.parse(opts.baseUrl, false, true).path || processModule.cwd();
+            const basepath = decodeURI(url.parse(opts.baseUrl, false, true).path || processModule.cwd());
```

Comment:
Probably can scope the decodeURI to the parse call instead of including the call to `cwd`